index.md

```diff
@@ -1,11 +1,18 @@
 ---
-title: Home
+title: Open Portable Trusted Execution Environment
```

Comment:
I think it would be better to say "OP-TEE - Open Portable Trusted Execution Environment" here. Since "OP-TEE" is what people refer to.

Reply:
When the page is rendered the title would be Open Portable Trusted Execution Environment - OP-TEE since the head.html include in theme concats the site.title and page.title

Reply:
Ok, then that'll do.

Reply:
Thanks @jbech-linaro !